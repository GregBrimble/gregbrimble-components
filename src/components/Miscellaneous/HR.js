import React from "react"
import styled from "styled-components"
import ThemeDefault from "../../themes/ThemeDefault"

const HR = styled(() => <hr align={`left`} />)`
  height: 6px;
  border: none;
  color: ${props => props.theme.supporting[0][`400`]};
  background-color: ${props => props.theme.supporting[0][`400`]};
  width: ${200 / 3}%;
`

HR.defaultProps = {
  theme: ThemeDefault,
}

export default HR
