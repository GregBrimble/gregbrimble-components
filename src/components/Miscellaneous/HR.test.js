import React from "react"
import HR from "./HR"
import renderer from "react-test-renderer"

describe("HR", () => {
  it("renders properly", () => {
    const tree = renderer.create(<HR />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
