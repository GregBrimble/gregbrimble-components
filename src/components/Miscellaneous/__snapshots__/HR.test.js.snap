// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR renders properly 1`] = `
<hr
  align="left"
/>
`;
